wiki/People/The_Team/Beatmap_Nominators/General_Information/en.md

```diff
@@ -24,9 +24,10 @@ Nominations and resetting nominations are the evidence of a Beatmap Nominator's
 
 ![disqualify and reset buttons](img/reset_dq.png)
 
-**This allows you to reset a beatmap's progress in the ranking proccess.** If a beatmap is currently nominated or qualified, you as a nominator are given the ability to reset the beatmaps' nomination(s) by posting a Problem stamp. If a beatmap is qualified, it can only be disqualified by a full Beatmap Nominator. These options are usually used when there is a major or unrankable issue in the beatmap which should be addressed before considering it for ranking.
+**This allows you to reset a beatmap's progress in the ranking proccess.** If a beatmap is currently nominated or qualified, you as a nominator are given the ability to reset the beatmaps' nomination(s) by posting a Problem stamp. If a beatmap is qualified, it can only be disqualified by a full BN. These options are usually used when there is a major or unrankable issue in the beatmap which should be addressed before considering it for ranking.
```

Comment:
```suggestion
**This allows you to reset a beatmap's progress in the ranking proccess.** If a beatmap is currently nominated or qualified, you as a nominator are given the ability to reset the beatmaps' nomination(s) by posting an issue using a Problem stamp. If a beatmap is qualified, it can only be disqualified by a full BN. These options are usually used when there is a major or unrankable issue in the beatmap which should be addressed before considering it for ranking.
```